tiatoolbox/utils/misc.py

```diff
@@ -805,21 +841,28 @@ def prepare_file_dir_cli(img_input, output_path, file_types, mode, sub_dirname):
 def prepare_model_cli(img_input, output_path, masks, file_types, mode):
     """Prepares cli for running models.
 
-    Checks for existing directories to run tests.
-    Converts file path to list of file paths or
-    creates list of file paths if input is a directory.
+    Checks for existing directories to run tests. Converts file path to
+    list of file paths or creates list of file paths if input is a
+    directory.
 
     Args:
-        img_input (str or pathlib.Path): file path to images.
-        output_path (str or pathlib.Path): output directory path.
-        masks (str or pathlib.Path): file path to masks.
-        file_types (str): file types to process using cli.
-        mode (str): wsi or tile mode.
+        img_input (str or pathlib.Path):
+            File path to images.
+        output_path (str or pathlib.Path):
+            Output directory path.
+        masks (str or pathlib.Path):
+            File path to masks.
+        file_types (str):
+            File types to process using cli.
+        mode (str):
+            One of "wsi" or "tile" mode.
 
     Returns:
-        files_all (list): list of file paths to process.
-        masks_all (list): list of masks corresponding to input files.
-        output_path (pathlib.Path): output path
+        list:
```

Comment:
give different names to the two output lists?

Reply:
Can you clarify whate you mean here?

Reply:
Should the two output lists not have different names?